booster/test/rpc-integration/test-implies2/state-antecedent-bottom.send

```diff
@@ -0,0 +1 @@
ag": "EVar", "name": "VarGENERATED'Unds'COUNTER'Unds'CELL", "sort": {"tag": "SortApp", "name": "SortGeneratedCounterCell", "args": []}}]}, {"tag": "And", "sort": {"tag": "SortApp", "name": "SortGeneratedTopCell", "args": []}, "patterns": [{"tag": "Equals", "argSort": {"tag": "SortApp", "name": "SortBool", "args": []}, "sort": {"tag": "SortApp", "name": "SortGeneratedTopCell", "args": []}, "first": {"tag": "DV", "sort": {"tag": "SortApp", "name": "SortBool", "args": []}, "value": "true"}, "second": {"tag": "App", "name": "Lbl'Unds-LT-'Int'Unds'", "sorts": [], "args": [{"tag": "DV", "sort": {"tag": "SortApp", "name": "SortInt", "args": []}, "value": "3"}, {"tag": "EVar", "name": "VarX", "sort": {"tag": "SortApp", "name": "SortInt", "args": []}}]}}, {"tag": "Equals", "argSort": {"tag": "SortApp", "name": "SortBool", "args": []}, "sort": {"tag": "SortApp", "name": "SortGeneratedTopCell", "args": []}, "first": {"tag": "DV", "sort": {"tag": "SortApp", "name": "SortBool", "args": []}, "value": "true"}, "second": {"tag": "App", "name": "Lbl'Unds-LT-'Int'Unds'", "sorts": [], "args": [{"tag": "EVar", "name": "VarX", "sort": {"tag": "SortApp", "name": "SortInt", "args": []}}, {"tag": "DV", "sort": {"tag": "SortApp", "name": "SortInt", "args": []}, "value": "3"}]}}]}]}}, "consequent": {"format": "KORE", "version": 1, "term": {"tag": "Exists", "sort": {"tag": "SortApp", "name": "SortGeneratedTopCell", "args": []}, "var": "VarY", "varSort": {"tag": "SortApp", "name": "SortAExp", "args": []}, "arg": {"tag": "App", "name": "Lbl'-LT-'generatedTop'-GT-'", "sorts": [], "args": [{"tag": "App", "name": "Lbl'-LT-'T'-GT-'", "sorts": [], "args": [{"tag": "App", "name": "Lbl'-LT-'k'-GT-'", "sorts": [], "args": [{"tag": "App", "name": "kseq", "sorts": [], "args": [{"tag": "App", "name": "inj", "sorts": [{"tag": "SortApp", "name": "SortPgm", "args": []}, {"tag": "SortApp", "name": "SortKItem", "args": []}], "args": [{"tag": "App", "name": "Lblint'UndsSClnUnds'", "sorts": [], "args": [{"tag": "App", "name": "Lbl'UndsCommUnds'", "sorts": [], "args": [{"tag": "DV", "sort": {"tag": "SortApp", "name": "SortId", "args": []}, "value": "$n"}, {"tag": "App", "name": "Lbl'UndsCommUnds'", "sorts": [], "args": [{"tag": "DV", "sort": {"tag": "SortApp", "name": "SortId", "args": []}, "value": "$s"}, {"tag": "App", "name": "Lbl'Stop'List'LBraQuotUndsCommUndsQuotRBra'", "sorts": [], "args": []}]}]}, {"tag": "App", "name": "Lbl'UndsEqlsUndsSCln'", "sorts": [], "args": [{"tag": "DV", "sort": {"tag": "SortApp", "name": "SortId", "args": []}, "value": "$n"}, {"tag": "EVar", "name": "VarY", "sort": {"tag": "SortApp", "name": "SortAExp", "args": []}}]}]}]}, {"tag": "App", "name": "dotk", "sorts": [], "args": []}]}]}, {"tag": "App", "name": "Lbl'-LT-'state'-GT-'", "sorts": [], "args": [{"tag": "App", "name": "Lbl'Stop'Map", "sorts": [], "args": []}]}]}, {"tag": "EVar", "name": "VarGENERATED'Unds'COUNTER'Unds'CELL", "sort": {"tag": "SortApp", "name": "SortGeneratedCounterCell", "args": []}}]}}}}}
```

Comment:
This is quite hard to read, maybe we should check in pretty-printed json?
Or, better, add a `README.md` file to the directory that contains a short description of what the data is?
For instance, this one here is not an obvious `Bottom` in the antecedent, what makes it `Bottom` is `X =Int 3` and `X <Int 3` in the constraints.